chain/x/zoracle/client/rest/types.go

```diff
@@ -17,22 +14,25 @@ type ScriptInfoWithTx struct {
 }
 
 type ReportDetail struct {
-	Reporter         sdk.ValAddress  `json:"reporter"`
-	TxHash           string          `json:"txhash"`
-	ReportedAtHeight int64           `json:"reportedAtHeight"`
-	ReportedAtTime   string          `json:"reportedAtTime"`
-	Value            json.RawMessage `json:"value"`
+	Reporter         sdk.ValAddress        `json:"reporter"`
+	TxHash           string                `json:"txhash"`
+	ReportedAtHeight int64                 `json:"reportedAtHeight"`
+	ReportedAtTime   string                `json:"reportedAtTime"`
+	Value            []types.RawDataReport `json:"value"`
 }
 
 type RequestQueryInfo struct {
```

Comment:
Let's call this RequestRESTInfo so we know it's the info for REST. We have way too many "info" structs lol.